unified_planning/grpc/unified_planning.proto

```diff
@@ -367,39 +367,43 @@ message Problem {
 enum Feature {
     // PROBLEM_CLASS
     ACTION_BASED = 0;
+    HIERARCHICAL = 1;
     // TIME
-    CONTINUOUS_TIME = 1;
-    DISCRETE_TIME = 2;
-    INTERMEDIATE_CONDITIONS_AND_EFFECTS = 3;
-    TIMED_EFFECT = 4;
-    TIMED_GOALS = 5;
-    DURATION_INEQUALITIES = 6;
+    CONTINUOUS_TIME = 2;
+    DISCRETE_TIME = 3;
+    INTERMEDIATE_CONDITIONS_AND_EFFECTS = 4;
+    TIMED_EFFECT = 5;
+    TIMED_GOALS = 6;
+    DURATION_INEQUALITIES = 7;
+    // EXPRESSION_DURATION
+    STATIC_FLUENTS_IN_DURATION = 8;
+    FLUENTS_IN_DURATION = 9;
     // NUMBERS
-    CONTINUOUS_NUMBERS = 7;
-    DISCRETE_NUMBERS = 8;
+    CONTINUOUS_NUMBERS = 10;
```

Comment:
The ids should not be changed as its breaks backward compatibility. They are only useful for compiling to the binary format but not visible by users.
So when adding a new field, you should just pick a new number that has not been previously and leave the other fields untouched.

Reply:
Ok, thanks for the feedback!